ydb/library/yql/yt/native/plugin.cpp

```diff
@@ -523,10 +589,12 @@ class TYqlPlugin
                 };
             }
             if (!ActiveQueriesProgress_[queryId].Compiled) {
+                ActiveQueriesProgress_.erase(queryId);
                 return TAbortResult{
                     .YsonError = MessageToYtErrorYson(Format("Query %v is not compiled", queryId)),
                 };
```

Comment:
Shouldn't we just return success in this case?
Otherwise it is strange that we return an error while aborting query. This behaviour is strange:
```
1. First abort -> ERROR Query is not compiled
2. Second abort -> ERROR Query is not found
```

```suggestion
                return {};
```